app/public/pos/create.js

```diff
@@ -0,0 +1,56 @@
+// rename the file to use across other files
+
+const catInput = document.getElementById("category");
+const nameInput = document.getElementById("name");
+const descInput = document.getElementById("description");
+const priceInput = document.getElementById("price");
+const submit = document.getElementById("submit");
+const message = document.getElementById("message");
+
+// display item_category
+fetch("/api/item/categories")
+    .then(response => {
+        return response.json();
+    })
+    .then(body => {
+        for (let i = 0; i < body.length; i++) {
+            const category = body[i];
+            const option = document.createElement("option");
+            option.value = category.id;
+            option.textContent = category.name;
+            catInput.append(option);
+        }
+    })
+    .catch(error => {
+        console.log(error);
+    });
+
+
+// handle item data sent to the backend
+submit.addEventListener("click", () => {
+    const item = {
+        category: catInput.value,
+        name: nameInput.value,
+        description: descInput.value,
+        price: priceInput.value
+    };
+
+    fetch("/api/item/add", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify(item),
+    })
+    .then(response => {
+        if (!response.ok) {
+            message.textContent = "Failure";
+        }
+        else {
+            message.textContent = "Success";
+        }
```

Comment:
You can use a ternary to shorten this down
```suggestion
        message.textContent = (response.ok ? "Success" : "Failure");
```